templates/github/.github/workflows/scripts/before_install.sh.j2

```diff
@@ -168,6 +168,11 @@ fi
 
 sed -i -e 's/DEBUG = False/DEBUG = True/' pulpcore/pulpcore/app/settings.py
 
+{% if not tasking_allow_async_unsafe -%}
+# Patch DJANGO_ALLOW_ASYNC_UNSAFE out of the pulpcore tasking_system
+# Don't let it fail. Be opportunistic.
+sed -i -e '/DJANGO_ALLOW_ASYNC_UNSAFE/d' ../pulpcore/pulpcore/tasking/entrypoint.py || true
```

Comment:
shouldn't it be done at `pulpcore/content/__init__.py` also?

Reply:
No, the content app is a different story...